tests/php/Service/RoomServiceTest.php

```diff
@@ -192,6 +192,7 @@ public function dataCreateConversationInvalidNames(): array {
 			[''],
 			['        '],
 			[str_repeat('a', 256)],
+			[str_repeat('😃', 200)],
```

Comment:
With the above change, this string is actually no longer invalid
But 
```
[str_repeat('‍💻', 200)],
```
is invalid as this emoji is a multibyte emoji (consists of 2 characters on database level)

Reply:
What we will do about this char?
```
php > echo mb_strlen(str_repeat('‍💻', 200));
400
```

Reply:
I put a comment in test about this case. Resolving this conversation because the code changed.